funnel/templates/macros.html.jinja2

```diff
@@ -706,3 +706,54 @@
     </div>
   </div>
 {%- endmacro %}
+
+{%- macro open_submission(project, showEditBtn=true) %}
+  <div class="flex-wrapper flex-wrapper--space-between flex-wrapper--start margin-top bottom-padding">
+    <div>
+      <div class="js-cfp-status {% if not project.cfp_state.OPEN -%} mui--hide {%- endif %}">
+        <p class="mui--text-subhead mui--text-bold margin-bottom" data-cy="cfp-state">{% trans %}Submission status: ON{% endtrans %}</p>
+        {% if not project.cfp_end_at_localized -%}
+          {% if not project.view_for('cfp').is_available() or not showEditBtn %}
+            <p class="mui--text-light mui--text-body2 zero-bottom-margin">{% trans %}Accepting submissions{% endtrans %}</p>
+          {% else %}
+            <p  class="mui--text-light mui--text-body2 zero-bottom-margin">{% trans %}Accepting submissions.{% endtrans %}<a href="{{ project.url_for('cfp') }}" class="mui--text-light mui--text-body2"> {% trans %}Click to pre-set an end date{% endtrans %}{{ faicon(icon='pencil', icon_size='caption', baseline=false, css_class='mui--text-light margin-left') }}</a></p>
```

Comment:
The space between sentences should be outside the `<a>` tag. Also, we may have a translation problem with the two sentences being in separate `{% trans %}` blocks, in case some language needs reverse order.

"Click to" is not required. Just "Set an end date" with a button, so it's not part of the sentence.